README.md

```diff
@@ -216,9 +247,32 @@ and bot users. This allows you to link as many channels as you would like with o
 
 See also https://github.com/matrix-org/matrix-appservice-bridge/blob/master/HOWTO.md for the general theory of all this :)
 
+------------
+
+#### Docker
+
+NOTE: The following instructions may be outdated. Be sure to inquire at the official
+[Matrix Bridging Room](https://matrix.to/#/#bridges:matrix.org) if you run into any problems
+
+Following the instructions above, generate a config and registration file. The file may also be hand-crafted if you're familiar with the layout. You'll need to place both these files in a "config" folder together to use the Docker image.
+
+```
+docker run -v path/to/config-folder:/usr/src/app/userconfig:z \
+			   cadair/matrix-appservice-slack:latest \
```

Comment:
These docker instructions would probably be better to build the dockerfile manually from the one in this repo, rather than use my rather unoffiicial one. That is until the official one is online.

Reply:
I copied the tutorial section from the discord bridge. Hm. I have no idea if it works...